test/executable_test.dart

```diff
@@ -13,60 +13,17 @@
 // limitations under the License.
 
 @TestOn('vm')
-
 import 'dart:io';
 
-import 'package:dependency_validator/dependency_validator.dart';
 import 'package:test/test.dart';
+import 'package:test_descriptor/test_descriptor.dart';
```

Comment:
It's a bit weird, but the convention for importing and using this package is to import it with a prefix of `d`. So far we've followed that convention in the few packages at workiva that use it so I think it'd be good to be consistent.